drivers/bluetooth/hci/h5.c

```diff
@@ -465,6 +466,15 @@ static void bt_uart_isr(const struct device *unused, void *user_data)
 				break;
 			}
 
+			header_checksum = ~((hdr[0] + hdr[1] + hdr[2]) & 0xff);
```

Comment:
Since both `hdr` and `header_checksum` are `uint8_t` the `& 0xff` actually does nothing here. 

Is it to make the calculation more recognizable from some specification?